grim.go

```diff
@@ -35,6 +37,14 @@ func (i *Instance) PrepareGrimQueue() error {
 		return fatalGrimErrorf("error while reading config: %v", err)
 	}
 
+	if config.origServerID != config.grimServerID {
+		if truncMessage == "" {
+			logger.Printf("GrimServerID shouldn't be over 15 characters and has been truncated\nPlease update your config.json file to have a shorter grimServerID\nOr remove the GrimServerID entry to use server defaults\n")
```

Comment:
Your error message refers to the id with 2 different proper nouns, one of which teh user cant/doesnt see.  I'd get rid of the last sentence as well.  The issue is that they might not have defined the GrimServerID at all, we could be defaulting it from some other value in their config file.

If you want to do the extra work and craft the error message to their specific case (ie you didn't define GrimServerID but did define GrimQueueName and it was too long or whatever) feel free.